packages/core/test/examples/12.test.ts

```diff
@@ -1,22 +0,0 @@
-import { pipe } from "@effect-ts/system/Function"
-
```

Comment:
Couldn't see any difference with example 10 besides InvariantT which, from what I understand, can be deleted with the new HKT system. (I'll test this assumption further though)